docs/tables/pagerduty_on_call.md

```diff
@@ -0,0 +1,30 @@
+# Table: pagerduty_on_call
+
+An on-call represents a contiguous unit of time for which a User will be on call for a given Escalation Policy and Escalation Rules.
+
+## Examples
+
+### Basic info
+
+```sql
+select
```

Comment:
@coop182 Could you please update the query format and remove the double quotes from the columns. Here is a [Sample](https://github.com/turbot/steampipe-plugin-pagerduty/blob/main/docs/tables/pagerduty_ruleset.md?plain=1) document for query structure.

Reply:
@bigdatasourav with "end" and "user" being reserved words - should I leave them in quotes?

Reply:
Interesting question @coop182 ... While we try to make the column names consistent with the API, in some cases it's better to actually do a different name to avoid conflict with the reserved words. In particular, I've found `user` to be problematic since it works and returns a value even when not quoted. (`end` is more likely to cause a query failure if entered without the double quotes.)

I think we should consider:
1. Calling it `user_on_call` (or similar) instead of `user`.
2. Keep `end`, but quote it in examples as you suggest.

What do you think?

CC @cbruno10 for his opinion as well.

Reply:
I'm happy with those suggestions... I will push them in.

Reply:
@e-gineer I'm aligned with those suggestions as well, moving away from `user`, especially if it doesn't error but returns an unexpected value will be helpful to users.